src/AwkwardArray.jl

```diff
@@ -1,19 +1,21 @@
-using JSON
+# using JSON
 
 module AwkwardArray
 
 ### Index ################################################################
 
-Index8 = AbstractArray{Int8,1}
-IndexU8 = AbstractArray{UInt8,1}
-Index32 = AbstractArray{Int32,1}
-IndexU32 = AbstractArray{UInt32,1}
-Index64 = AbstractArray{Int64,1}
+Index8 = AbstractVector{Int8}
+IndexU8 = AbstractVector{UInt8}
+Index32 = AbstractVector{Int32}
+IndexU32 = AbstractVector{UInt32}
+Index64 = AbstractVector{Int64}
 IndexBig = Union{Index32,IndexU32,Index64}
```

Comment:
```suggestion
const Index8 = AbstractVector{Int8}
const IndexU8 = AbstractVector{UInt8}
const Index32 = AbstractVector{Int32}
const IndexU32 = AbstractVector{UInt32}
const Index64 = AbstractVector{Int64}
const IndexBig = Union{Index32,IndexU32,Index64}
```

for type aliases, you need `const` to maintain type stability (otherwise compiler would worry `AwkwardArray.IndexBig` gets changed mid-execution)